tests/unit/stats/exporter/test_prometheus_stats.py

```diff
@@ -37,44 +37,47 @@
     "my.org/measure/video_size_test2", "size of processed videos", "By")
 
 VIDEO_SIZE_MEASURE_FLOAT = measure_module.MeasureFloat(
-    "my.org/measure/video_size_test-float", "size of processed videos-float", "By")
+    "my.org/measure/video_size_test-float", "size of processed videos-float",
+    "By")
 
 VIDEO_SIZE_VIEW_NAME = "my.org/views/video_size_test2"
 VIDEO_SIZE_DISTRIBUTION = aggregation_module.DistributionAggregation(
-                            [0.0, 16.0 * MiB, 256.0 * MiB])
-VIDEO_SIZE_VIEW = view_module.View(VIDEO_SIZE_VIEW_NAME,
-                                "processed video size over time",
-                                [FRONTEND_KEY],
-                                VIDEO_SIZE_MEASURE,
-                                VIDEO_SIZE_DISTRIBUTION)
-REGISTERED_VIEW = {'test1_my.org/views/video_size_test2-my.org/keys/frontend':
-                       {'documentation': 'processed video size over time',
-                        'labels': ['my.org/keys/frontend'],
-                        'name': 'test1_my.org/views/video_size_test2'}}
-
-REGISTERED_VIEW2 = {'opencensus_my.org/views/video_size_test2-my.org/keys/frontend':
-                        {'documentation': 'processed video size over time',
-                         'labels': ['my.org/keys/frontend'],
-                         'name': 'opencensus_my.org/views/video_size_test2'}}
+    [16.0 * MiB, 256.0 * MiB])
```

Comment:
This is the important change, most of the rest below is lint.

Reply:
I would do this (lint) changes in separate PR. If you dont mind can you send another PR for lint styling?

Reply:
I moved the lint changes to https://github.com/census-instrumentation/opencensus-python/pull/390, should clean this diff up once it's merged.